src/main/resources/default/taglib/t/multiSelect.html.pasta

```diff
@@ -23,6 +24,9 @@
     <i:if test="isFilled(label)">
         <label class="form-label"><span class="@labelClass">@label</span></label>
     </i:if>
```

Comment:
I think a more flexibel way would be to define a `<i:render name="label>` containing the current label-rendering as a default and being able to override it where necessary via `<i:block>`

Reply:
You could use `addon` of `t:textfield` as inspiration